pip/tests-integration/interop_qiskit/test_circuits/test_circuits.py

```diff
@@ -0,0 +1,683 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import pytest
+
+import numpy as np
+from typing import Tuple, List
+
+from interop_qiskit import QISKIT_AVAILABLE, SKIP_REASON
+
+if QISKIT_AVAILABLE:
+    from qiskit.circuit import QuantumCircuit
+
+
+def get_parameterized_circuit(num_qubits: int) -> "QuantumCircuit":
```

Comment:
This function doesn't seem to be called explicitly anywhere. Am I missing something magical?